plutus-core/plutus-ir/src/PlutusIR/Subst.hs

```diff
@@ -129,7 +129,6 @@ funRes f = \case
   TyFun a dom cod -> TyFun a dom <$> funRes f cod
   t               -> f t
 
--- TODO: these could be Traversals
 -- | Get all the term variables in a term.
 vTerm :: Fold (Term tyname name uni fun ann) name
 vTerm = termSubtermsDeep . termVars
```

Comment:
I think I tried it, but I was hit but `cosmosOf`.  CosmosOf requires Contravariant f, which Traversals cannot provide

Reply:
I mean, is it in principle possible to have a `Traversal` focused on all the variables of a term? I think it is. If `cosmosOf` is getting in the way, then it just means `cosmosOf` isn't suitable for the job and we should use something else. Maybe we need to define this one in terms of `deepOf` or something, I'm not intimately familiar with `lens`.

Reply:
I do think it is possible, it's just we don't crucially need atm. Also it can be misused and lead to more variable capturing problems? Anyway, I will bring back the note in another PR but with a MAYBE tag.